deploy/materialized_view_address.sql

```diff
@@ -4,13 +4,7 @@
 begin;
 
 create materialized view ggircs_swrs.address as (
-  with x as (
-    select _ghgr_import.xml_file as source_xml,
-           _ghgr_import.id         as ghgr_import_id
-    from ggircs_swrs.ghgr_import as _ghgr_import
-    order by ghgr_import_id asc
-  )
-  select ghgr_import_id,
+  select id as ghgr_import_id,
```

Comment:
Same id conflict with table exporter, this will be true of any materialized view deploy file